src/strings.bif

```diff
@@ -1527,6 +1526,7 @@ function rjust%(str: string, width: count, fill: string &default=" "%) : string
 function swap_case%(str: string%) : string
 	%{
 	string s = str->ToStdString();
+	// NOLINTNEXTLINE(modernize-loop-convert)
```

Comment:
Is this modern enough?
```
--- a/src/strings.bif
+++ b/src/strings.bif
@@ -1527,12 +1527,12 @@ function rjust%(str: string, width: count, fill: string &default=" "%) : string
 function swap_case%(str: string%) : string
        %{
        string s = str->ToStdString();
-       for ( size_t i = 0; i < s.size(); i++ )
+       for ( auto& c : s )
                {
-               if ( std::islower(s[i]) )
-                       s[i] = toupper(s[i]);
-               else if ( std::isupper(s[i]) )
-                       s[i] = tolower(s[i]);
+               if ( std::islower(c) )
+                       c = toupper(c);
+               else if ( std::isupper(c) )
+                       c = tolower(c);
                }
```

Reply:
None of this is safe if the character is not ASCII. All the values here are `char` on my platform while all these C functions expect `unsigned char`. IIRC passing them anything which is not equal `EOF` and doesn't fit into `unsigned char` triggers UB.

Reply:
That it was here before doesn't mean was good.